test/sqllogictest/transform/predicate_pushdown.slt

```diff
@@ -311,3 +311,93 @@ Query:
 | | implementation = Differential %1 %0.((#0 + 1))
 
 EOF
+
+# Join-Dependent Predicate Duplication (JoinInputMapper::consequence_for_input)
+
+query T multiline
+EXPLAIN
+SELECT * FROM t1, t2 WHERE t1.f1 = t2.f1 AND ((t1.f2 = 3 AND t2.f2 = 4) OR (t1.f2 = 5 AND t2.f2 = 6));
```

Comment:
You might as well use the new explain interface to print out these plans to save effort when switching between old and new explain.

Reply:
I guess the entire `predicate_pushdown.slt` will be rewritten using `--rewrite-results`, so the new tests won't really add extra effort.